apps/api/src/lib/flat-parse.ts

```diff
@@ -1,118 +1,52 @@
 import { z } from "zod/v4";
 
-export type { FlatParseResult };
-export { flatParse };
+import { ApiError } from "@recallnet/services/types";
```

Comment:
For some reason, this dependency feels weird to me. A lib, which is a simple helper function, depending on a type of services package. But okay, not really sure what would be a better way.